ui/k-toolbar/main.kytos

```diff
@@ -269,6 +354,15 @@
                 componentKey: "",
                 show_post_modal: false,
                 modal_message_post: "Do you want to create a maintenance window with NO End Time?",
+                time_date: true,
```

Comment:
@Alopalao could you give a more descriptive name for this boolean var? Maybe something like `using_relative_date` or `has_relative_date` or something along these lines? I think it'd be more readable in the conditionals.

Reply:
Changed to `using_relative_date`